experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -238,10 +238,8 @@ if energy_check && cs.mode.name == "slabplanet"
     )
 end
 
-@show cs.fields.P_liq
 # # animations
 if parsed_args["anim"]
-    #make it so this works with slab land?
     include("coupler_utils/viz_explorer.jl")
     plot_anim(
```

Comment:
would it make more sense to pass in `cs` directly, and unpack inside `plot_anim`, or do we want to pass this longer list of arguments (e.g. because we will dispatch off of the argument types later)?

Reply:
Sounds good, we can collapse for now to make it more concise, and then expand if needs be! :)